File: app/src/main/java/hk/collaction/timber/model/TreePhotoPath.java
package hk.collaction.timber.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Himphen on 23/8/2017.
 */

public class TreePhotoPath {
	@SerializedName("filename")
	private String filename;

	@SerializedName("original")
	private String original;

	public String getFilename() {
		return filename;
	}

	public String getOriginal() {
		return original;
	}
}

File: app/src/main/java/hk/collaction/timber/api/model/TreePhotoPath.kt
package hk.collaction.timber.api.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class TreePhotoPath(
    @SerializedName("filename") val filename: String? = null,
    @SerializedName("original") val original: String? = null
) : Parcelable